sdk/storage/Azure.Storage.DataMovement.Blobs/tests/DataMovementBlobTestBase.cs

```diff
@@ -0,0 +1,562 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Storage.Test.Shared;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using System.Net;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage.Test;
+using System.IO;
+using NUnit.Framework;
+using Azure.Core;
+using System.Threading;
+using Azure.Storage.DataMovement.Tests;
+using Azure.Storage.Blobs;
+
+namespace Azure.Storage.DataMovement.Blobs.Tests.Shared
+{
+    /// <summary>
+    /// Base class for Common tests
+    /// </summary>
+    [ClientTestFixture(
+    BlobClientOptions.ServiceVersion.V2020_06_12,
+    BlobClientOptions.ServiceVersion.V2020_08_04,
+    RecordingServiceVersion = BlobClientOptions.ServiceVersion.V2020_08_04,
+    LiveServiceVersions = new object[] { BlobClientOptions.ServiceVersion.V2020_06_12 })]
+    public abstract class DataMovementBlobTestBase : DataMovementTestBase
```

Comment:
Where are the actual tests for DM lib?

Reply:
In the Azure.Storage.DataMovement/tests folder. not this one. This one just has the storage resource specific to the blobs package.